Favor/Favor/Sources/Scenes/FriendList/FriendSection.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  FriendSection.swift
+//  Favor
+//
+//  Created by 이창준 on 2023/04/25.
+//
+
+import UIKit
+
+import FavorKit
+
+public enum FriendSectionItem: SectionModelItem {
+  case friend(Friend)
+}
+
+public enum FriendSection: SectionModelType {
+  case friend
+  case editFriend
+}
+
+extension FriendSectionItem: Equatable, Hashable {
+  public static func == (lhs: FriendSectionItem, rhs: FriendSectionItem) -> Bool {
+    switch (lhs, rhs) {
+    case let (.friend(lhsValue), .friend(rhsValue)):
+      return lhsValue.friendNo == rhsValue.friendNo
+    }
+  }
```

Comment:
`Hashable` 안에 `Equatable`이 이미 채택되어 있기 때문에
`Equatable`은 지워도 될 것 같아 !

Reply:
아 그러네..!

`SectionModelItem`이 `Hashable`을 채택한 프로토콜이니까 그냥 아예 뺄게..!

```suggestion
extension FriendSectionItem {
  public static func == (lhs: FriendSectionItem, rhs: FriendSectionItem) -> Bool {
    switch (lhs, rhs) {
    case let (.friend(lhsValue), .friend(rhsValue)):
      return lhsValue.friendNo == rhsValue.friendNo
    }
  }
```